app/assets/javascripts/govuk_publishing_components/lib/govspeak/magna-charta.js

```diff
@@ -458,12 +456,12 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     }
   }
 
-  MagnaCharta.prototype.getChartId = function (module) {
+  MagnaCharta.prototype.getChartId = function () {
     var allCharts = document.querySelectorAll('table.js-barchart-table')
     var id = null
 
     for (var i = 0; i < allCharts.length; i++) {
-      if (allCharts[i] === module) {
+      if (allCharts[i] === this.$module) {
```

Comment:
was this meant to be `if (allCharts[i] === this.$table) {`? Haven't expanded the two code, but judging by the above changes we don't seem to have a `this.$module` anymore

Reply:
Yes, I think so. Good spot, thanks, will change before I merge 👍

Reply:
Slightly worrying that this didn't cause a test failure.